tests/elections/test_election.py

```diff
@@ -1,64 +1,143 @@
-from unittest.mock import MagicMock
-
 import pytest
 
+from tests.utils import generate_election, generate_validators
+
+from bigchaindb.lib import Block
 from bigchaindb.elections.election import Election
+from bigchaindb.migrations.chain_migration_election import ChainMigrationElection
+from bigchaindb.upsert_validator.validator_election import ValidatorElection
 
 
 @pytest.mark.bdb
-def test_approved_elections_one_migration_one_upsert(
-        b,
-        ongoing_validator_election, validator_election_votes,
-        ongoing_chain_migration_election, chain_migration_election_votes
-):
-    txns = validator_election_votes + \
-           chain_migration_election_votes
-    mock_chain_migration, mock_store_validator = run_approved_elections(b, txns)
-    mock_chain_migration.assert_called_once()
-    mock_store_validator.assert_called_once()
+def test_approved_elections_concludes_all_elections(b):
```

Comment:
I would also suggest writing a test wherein the order of validator election and migration witnessed is reversed. This will allow us to assert the rule that a block with validator and migration election concludes both regardless of their order.

Reply:
Good point! But I would do it in the next PR where I implement [this](https://github.com/bigchaindb/BEPs/pull/82#discussion_r218057405).